.github/linters/.flake8

```
@@ -1,4 +1,4 @@
 [flake8]
 # https://black.readthedocs.io/en/stable/compatible_configs.html#flake8
-extend-ignore = E203,W503
+extend-ignore = E203,W503,F401
```

I'm curious about why you are ignoring the error about unused import :)

---

it's complaining about a unused import that i use to fallback to Python2 ( i don't know still if ARGO has moved to py3).

---

But are you using this import somewhere? I don't need any reference to using urlparse anywhere.
If not, what's the purpose?

---

argh, i have removed the code using urlparse cause it was not needed:-) let me fix it..

---

fixed